File: app/src/androidTest/java/com/example/benchmarks/models/benchmark/TestOperationsCollections.java
package com.example.benchmarks.models.benchmark;

public class TestOperationsCollections extends OperationsCollections {
    @Override
    public long measureTime(int sizeOfCollection, BenchmarkItem item) {
        try {
            Thread.sleep(500);
        } catch (InterruptedException e) {
            e.printStackTrace();
        }
        return 100L;
    }
}

File: app/src/androidTest/java/com/example/benchmarks/models/benchmark/TestOperationsCollections.kt
package com.example.benchmarks.models.benchmark

class TestOperationsCollections : OperationsCollections() {
    override fun measureTime(sizeOfCollection: Int, item: BenchmarkItem): Long {
        try {
            Thread.sleep(Tests.DELAY)
        } catch (e: InterruptedException) {
            e.printStackTrace()
        }
        return Tests.MEASURE_TIME
    }
}